### PR TITLE
Create the administrator transfer page

### PR DESCRIPTION
**Pull request by Luiz-FS**
**Feature/Bug description:** Create the administrator transfer page so that an administrator can move his position to another member of the institution.

**Solution:** Using the angular directives and the layout created by the design, I created the admin transfer page and the dialog to search for the member to whom the invitation will be sent.

screenshot from 2018-03-14 11-26-16
screenshot from 2018-03-14 11-26-54
screenshot from 2018-03-14 11-27-07
screenshot from 2018-03-14 11-28-23
screenshot from 2018-03-15 10-37-11

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/notification/notificationService.js [hunk 1]
```diff
@@ -41,7 +41,8 @@
             'ACCEPTED_LINK': messageCreator('Aceitou sua solicitação de vínculo com ', SINGLE_INST),
             'SHARED_EVENT': messageCreator('Compartilhou um evento de ', SINGLE_INST),
             'DELETED_POST': messageCreator('Deletou o seu post em ', SINGLE_INST),
-            'REPLY_COMMENT': messageCreator('Respondeu um comentário seu')
+            'REPLY_COMMENT': messageCreator('Respondeu um comentário seu'),
+            'USER_ADM': messageCreator('Enviou um convite para ser administrador da instituição ', SINGLE_INST)
```
Comment (mayzabeel) [inline frontend/notification/notificationService.js]: Indicou você para ser administrador da instituição...

File: frontend/institution/transfer_admin_dialog.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            </md-list-item>
+        </md-list>
+    </div>
+ </md-dialog-content>
+ <md-dialog-actions>
+     <md-button class="md-raised" md-colors="{color: 'default-teal-500'}" ng-click="transferAdminCtrl.cancel()">
+        CANCELAR
+     </md-button>
+     <md-button class="md-raised" md-colors="{color: 'default-teal-500'}" ng-click="transferAdminCtrl.confirm()">
+        COMFIRMAR
```
Comment (mayzabeel) [inline frontend/institution/transfer_admin_dialog.html]: Confirmar

File: frontend/institution/transfer_admin_dialog.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                ng-click="transferAdminCtrl.selectMember(member)" flex-xs>
+                <img title="{{member.name}}" ng-src="{{member.photo_url}}" class="md-avatar" style="margin: 0 15px 0 -15px; width: 60px; height: 60px;"/>
+                <div layout="column">
+                    <b style="position: relative; margin-top: 20px; font-size: 15px;">{{ member.name }}</b>
+                    <span style="position: relative; bottom: 20px; font-size: 15px;">{{ member.email[0]}}</span>
+                </div>
+            </md-list-item>
+            <md-list-item ng-if="members.length === 0">
+                <div layout="row" layout-align="center" flex="100">
+                    <p>Não existe membro com esse nome ou email</p>
```
Comment (mayzabeel) [inline frontend/institution/transfer_admin_dialog.html]: Nenhum membro foi encontrado com esse nome ou email.

File: frontend/institution/management_members.html [hunk 4]
```diff
@@ -60,6 +60,70 @@ <h2 md-colors="{color: 'teal-900'}">Convite em lote</h2>
...
+                  class="md-avatar" style="margin: 0 15px 0 px; width: 60px; height: 60px;"/>
+                <div layout="column">
+                  <b style="position: relative; margin-top: 20px; font-size: 15px;">{{ manageMemberCtrl.getMemberName(invite.invitee_key) }}</b>
+                  <span style="position: relative; bottom: 20px; font-size: 15px;">{{ invite.invitee }}</span>
+                </div>
+                <div layout="row" layout-align="end center" flex>
+                  <p style="border: solid; height: 35px; width: 148px; margin-right: -6px;
+                    text-align: center; border-color: rgb(0, 150, 136); border-radius: 3px;
+                    color: rgb(0, 150, 136); font-weight: bold; line-height: 21px; padding: 5px;"
+                    ng-if="invite.status === 'sent'">EM APROVAÇÃO</p>
```
Comment (mayzabeel) [inline frontend/institution/management_members.html]: Pendente

File: frontend/institution/managementMembersController.js [hunk 5]
```diff
@@ -304,6 +331,29 @@
...
+            let member = getMemberByKey(key);
+            return member.photo_url || '/app/images/avatar.png';
+        };
+
+        manageMemberCtrl.getMemberName = function getMembeName(key) {
+            let member = getMemberByKey(key);
+            return member.name || 'Nome do Membro';
+        };
+
+        manageMemberCtrl.disableTransferAdminButton = function disableTransferAdminButton() {
```
Comment (mayzabeel) [inline frontend/institution/managementMembersController.js]: teste

File: frontend/institution/managementMembersController.js [hunk 6]
```diff
@@ -428,4 +478,56 @@
             return Utils.validateEmail(email);
         }
     });
+
+    app.controller('TransferAdminController', function(institution_members, institution, InviteService, MessageService, $mdDialog) {
```
Comment (mayzabeel) [inline frontend/institution/managementMembersController.js]: Teste

File: frontend/institution/management_members.html [hunk 7]
```diff
@@ -60,6 +60,77 @@ <h2 md-colors="{color: 'teal-900'}">Convite em lote</h2>
...
+                <img title="{{manageMemberCtrl.institution.admin.name}}" ng-src="{{manageMemberCtrl.getMemberPhotoUrl(invite.invitee_key)}}" 
+                  class="md-avatar" style="margin: 0 15px 0 px; width: 60px; height: 60px;"/>
+                <div layout="column">
+                  <b style="position: relative; margin-top: 20px; font-size: 15px;">{{ manageMemberCtrl.getMemberName(invite.invitee_key) }}</b>
+                  <span style="position: relative; bottom: 20px; font-size: 15px;">{{ invite.invitee }}</span>
+                </div>
+                <div layout="row" layout-align="end center" flex>
+                  <p style="border: solid; height: 35px; width: 148px; margin-right: -6px;
+                    text-align: center; border-color: rgb(0, 150, 136); border-radius: 3px;
+                    color: rgb(0, 150, 136); font-weight: bold; line-height: 21px; padding: 5px;"
```
Comment (tiagolimpe) [inline frontend/institution/management_members.html]: This style is equal to the style of line 89. Extract to a CSS class.

File: backend/models/invite_user_adm.py [hunk 8]
```diff
@@ -16,6 +17,17 @@ def check_invite(self):
             "The invitee is not a member of this institution!", 
             NotAuthorizedException)
         
+        queryAnotherInvite = InviteUserAdm.query(
```
Comment (mayzabeel) [inline backend/models/invite_user_adm.py]: Check if invite sender is admin

**Pull request merged**